src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const editWords = (words) => words.trim();
+  const editArray = sourceString
+    .split(';')
+    .filter((el) => el !== '' && el !== '  ;')
+    .map((arr) => arr.split(':'))
+    .map((arr) => arr.map(editWords));
```

Comment:
[CODE STYLE] - The chained methods should have the same indentation to improve readability.